### PR TITLE
Csdp git source UI

### PR DESCRIPTION
**Pull request by NimRegev**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/csdp-git-source-ui/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/csdp-git-source-ui/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/csdp-git-source-ui/docs/

File: _docs/runtime/git-sources.md [hunk 1]
```diff
@@ -43,9 +50,43 @@ Git Sources are displayed when you drill down on a runtime.
...
+	file="/images/runtime/edit-git-source.png" 
+	url="/images/runtime/edit-git-source.png" 
+	alt="Edit Git Source" 
+	caption="Edit Git Source"
+    max-width="30%" 
+%}
+{:start="4"}
+1. Change the **Source** and **Destination** definitions for the Git Source, and select **Save**. 
 
 ### What to read next
```
Comment (kostis-codefresh) [inline _docs/runtime/git-sources.md:90]: What do I do with the git source after it is created? Where can I use it?

Comment (NimRegev) [reply _docs/runtime/git-sources.md:90]: Once you create a Git Source for a runtime, it is available for selection to store resources for a CI/CD entity assocaited with that runtime . For example, when you create pipelines or applications, you can select the Git Source in which to store manifest definitions.

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/csdp-git-source-ui/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/csdp-git-source-ui/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://cap-staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/csdp-git-source-ui/docs/

**Pull request merged**